kpi/views/v2/asset.py

```diff
@@ -618,6 +620,68 @@ def metadata(self, request):
         metadata = self.get_metadata(queryset)
         return Response(metadata)
 
+    @action(detail=False, methods=['GET'],
+            renderer_classes=[renderers.JSONRenderer],
+            url_path=r'migrate(?:/(?P<task_id>[\d\w\-]+))?')
+    def migrate(self, request, task_id: str = None, **kwargs):
+        """
+
+        Really basic endpoint what allows super users to migration projects
+        from KoBoCAT.
+
+        1. Call this endpoint with `?username=<username>`
+        2. Fetch url provided to check the state of the Celery task.
+           It can be:
+            - 'PENDING'
+            - 'FAILED'
+            - 'SUCCESS'
+
+        """
+        if not request.user.is_superuser:
+            raise Http404
+
+        if task_id:
+            from celery.result import AsyncResult
+            res = AsyncResult(task_id)
+            if res:
+                return Response(
+                    {
+                        'status': res.state
+                    },
+                )
```

Comment:
All hail our new god, psf/black? Yikes. Can't say I like this formatting at all.

Reply:
Oops, no it should have been on one line. I was testing a new shortcut to enable `Black` on those lines. I forgot to revert it.